website/docs/getting-started/quickstart.mdx

```diff
@@ -87,7 +85,7 @@ To add support for the LIGO smart contract language, install the LIGO plugin by
 taq install @taqueria/plugin-ligo
 ```
 
-Once installed, the plugin provides the task `taq compile` which when run, will look for any LIGO files in the `/contracts` directory and compile them to Michelson `.tz` files in the `/artifacts` directory. The plugin also provides a template to easily create and register a Ligo contract using the `taq create contract taks
+Once installed, the plugin provides the task `taq compile` which when run, will look for any LIGO files in the `/contracts` directory and compile them to Michelson `.tz` files in the `/artifacts` directory. The plugin also provides a template to easily create and register a Ligo contract using the `taq create contract task
```

Comment:
Same missing "\`" as the root `quickstart.md`

Reply:
The compile task behaves a little different now. `taq compile` will compile any registered contracts found in the `contracts` directory, unless you run `taq compile [sourceFile]`.